components/nexus-repository/src/main/java/org/sonatype/nexus/repository/proxy/ProxyFacetSupport.java

```diff
@@ -86,6 +86,8 @@ public String toString() {
 
   private boolean remoteUrlChanged;
 
+  private String cacheToken;
```

Comment:
@mcculls No, nothing yet. The "concurrency" issue here is that ongoing content request might still be treated as non-expired, but unsure how "strict" we want to be regarding expiring caches. IMHO, this is a non-issue, but we can add "volatile" at least if we want to.